xcm/src/v3/mod.rs

```diff
@@ -340,7 +340,9 @@ impl TryFrom<OldWeightLimit> for WeightLimit {
 #[derive(Clone, Eq, PartialEq, Encode, Decode, Debug)]
 pub struct XcmContext {
 	/// The `MultiLocation` origin of the corresponding XCM.
-	pub origin: Option<MultiLocation>,
+	/// The origin is could point to physical locations (Chains, etc),
+	/// but also conceptual locations such as Accounts or Pallet.
```

Comment:
This is not what a computed origin is: check the docs here for info: https://github.com/paritytech/polkadot/blob/254c521db67ab9562eb2b0463cb2091211dde07c/xcm/xcm-builder/src/barriers.rs#L115-L118

Reply:
I'm not sure how to put it in words as a definition. Do you have any suggestions?